src/sas/qtgui/Plotting/ColorMap.py

```diff
@@ -154,27 +154,33 @@ def initRangeSlider(self):
         """
         Create and display the double slider for data range mapping.
         """
-        self.slider = RangeSlider()
-        self.slider.setMinimum(self.vmin)
-        self.slider.setMaximum(self.vmax)
-        self.slider.setLowValue(self.vmin)
-        self.slider.setHighValue(self.vmax)
-        self.slider.setOrientation(QtCore.Qt.Horizontal)
+        self.slider = QDoubleRangeSlider(QtCore.Qt.Horizontal)
+        self.slider.setValue([self.vmin, self.vmax])
+        self.slider.setRange(self.vmin, self.vmax)
+
+        self.slider.setTickPosition(QtWidgets.QSlider.TicksAbove)
+        self.slider.setSliderPosition([self.vmin, self.vmax])
 
         self.slider_label = QtWidgets.QLabel()
         self.slider_label.setText("Drag the sliders to adjust color range.")
 
         def set_vmin(value):
             self.vmin = value
-            self.txtMinAmplitude.setText(str(value))
+            self.txtMinAmplitude.setText(formatNumber(value))
             self.updateMap()
         def set_vmax(value):
             self.vmax = value
-            self.txtMaxAmplitude.setText(str(value))
+            self.txtMaxAmplitude.setText(formatNumber(value))
             self.updateMap()
 
-        self.slider.lowValueChanged.connect(set_vmin)
-        self.slider.highValueChanged.connect(set_vmax)
+        def set_values(values):
```

Comment:
why not set_values(v1: int/float, v2: int/float), makes it easier to see what it needs

Reply:
or Tuple[int/float, int/float] if it needs to be a tuple